infra/gcp/terraform/db-manager.tf

```diff
@@ -64,6 +72,10 @@ resource "google_cloud_run_service" "db_manager" {
           name  = "NODE_ENV"
           value = "production"
         }
+        env {
+          name  = "AUTH0_DOMAIN"
+          value = "dummy"
+        }
```

Comment:
same comments as in the backend tf file

Reply:
See [comment below](https://github.com/nearform/titus/pull/795#issuecomment-811709476), need to leave this in place.